src/index.css

```diff
@@ -8,7 +8,7 @@
 }
 
 body {
-  background-color: #ec448a;
+  overflow-x: hidden;
 }
 
 h1 {
```

Comment:
- [ ] Good job trying to implement the previous reviewer changes request +1:  But Please try there are still some differences between your design and the original design :cry:  Please follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) That means include typography: font face, size, and weight and have the same Layout: composition and space between elements. :+1: Beacause as developer It's very important to pay attention to details :smile: 

> Please try to make almost an exact copy of the given design. It's a key part of this project and a major requirement to pass the task.
> 

#### Your design
![Screenshot from 2022-09-13 21-46-39](https://user-images.githubusercontent.com/6615394/190015267-940f2620-03b3-4397-a0ff-7e2da197ef9c.png)

#### Original design
![189962427-91e660f7-2f4f-41cd-aa2a-7ef4e9eb9301](https://user-images.githubusercontent.com/6615394/190015938-ae36875a-93b7-4edf-bc18-d7eedee1f5f6.png)

I would like to recommend you to check this [project](https://covid19-metrics-app-aka.netlify.app/) :smile: 

- [ ] [OPTIONAL] Please don't forget to always deploy your application with the change you made . It's very importante to always update your live version. :+1:

Reply:
@codecaiine Thank you for your feedback to make my code better. I like my final current work way more after implementing the changes. A quick question on deployment, I deployed the dev branch. When making a pull request I make it on a new branch. Can I deploy before merging to the dev branch?

Reply:
Well resolved 👍🏽